glutil/partitioner.py

```diff
@@ -268,17 +270,17 @@ def missing_partitions(self):
         return missing
 
     def delete_partitions(self, partitions_to_delete):
-        """Remove partitions from the Glue database
+        """Remove partitions from the Glue Data Catalog
 
         Args:
-            partitions_to_delete (list): A list of Partitions to remove from
-                the Glue database
+            partitions_to_delete (list): A list of partitions to remove from
```

Comment:
With the above, I think this should stay `Partition`, because it requires `Partition` objects. If someone passed in a list of partition values (ex `[['2019', '01', '02', '03'], ...]`) this method wouldn't work.